src/components/07-form/controls/date-input.njk

```diff
@@ -1,18 +1,20 @@
-<fieldset>
-  <legend>Date of birth</legend>
-  <span class="usa-form-hint" id="dobHint">For example: 04 28 1986</span>
-  <div class="usa-date-of-birth">
-    <div class="usa-form-group usa-form-group-month">
-      <label for="date_of_birth_1">Month</label>
-      <input class="usa-input-inline" aria-describedby="dobHint" id="date_of_birth_1" name="date_of_birth_1" type="number" min="1" max="12" value="">
+<form>
```

Comment:
Adding a form element was the only way to test this, i.e. to get the autofill to show up on the fractal site. Do we want to keep this in or remove?